data/water/gsl-water-level-2016/index.html

```diff
@@ -0,0 +1,55 @@
+---
+author:
+  display_name: UGRC Staff
+  email: ugrc@utah.gov
+tags:
+  - history
+  - lakes
+  - water
+  - gsl
+categories: []
+date: 2022-05-03 12:00:00 -0600
+GSL2016:
+  hub:
+    name: Utah Great Salt Lake Water Level 2016
+    item_id: NEED THIS
+  updates:
+    - 2016
+---
+<figure class="caption caption--right">
+ <a href="{% link images/GSL2016_lg.png %}">
+  <img class="caption__image" title="map" src="{% link images/GSL2016.png %}"
+    alt="UtahGreatSaltLakeWaterLevel2016" loading="lazy" /></a>
+  <figcaption class="caption__text">GSL Water Level 2016</figcaption>
+</figure>
+{% include abstract.html
+name="Great Salt Lake Resources"
+stewards="UGRC"
+abstract="Various resources related to the Great Salt Lake." %}
+<div class="grid package">
+  <div class="grid__col grid__col--12-of-12">
+    <h3 id="GSL2016">Utah Great Salt Lake Water Level 2016</h3>
+  </div>
+  <div class="grid__col grid__col--12-of-12 package-content">
+    <p>This dataset was derived from the 
+    <a href="https://storage.googleapis.com/state-of-utah-sgid-downloads/lidar/great-salt-lake-2016/GSL_shps.zip">breakline features</a> 
+    associated with the <a href="{% link data/elevation-and-terrain/2016-lidar-gsl/index.md %}">2016 Great Salt Lake LiDAR project</a>. 
+    To create this dataset the <i>Lake Pond</i> features were extracted from the <i>GreatSaltLake_Breaklines</i> shapefile and converted 
+    into polygons. All polygons outside of the main north and south arms of the lake were removed. The <i>Elevation_m</i> & <i>Elevation_ft</i> 
```

Comment:
**question**: weren't the attributes named elev_m and elev_ft?

Reply:
actual field name vs. alias and I chose to use the alias. 21st century software defaults to display the alias if it exist I believe. But I could go either way

Reply:
it will only show the alias if consumed in an esri product from the arcgis online feature service or the file geodatabase download right? Every other combination will only have the field name?

Reply:
probably something like that. the page now has both